### ISSUE TITLE
Replace menu should be keyboard accessible

### ISSUE THREAD

**Issue by barmac** [bpmn-io/bpmn-js-connectors-extension#30]
__Is your feature request related to a problem? Please describe__

When I activate the replace menu, it shows the first element as selected. However, when I press arrow keys, it doesn't change the selected replacement but rather moves the element.

https://user-images.githubusercontent.com/28307541/160992245-7a63969f-d7b7-4d18-83b0-191dde73509a.mov

__Describe the solution you'd like__

Keyboard focus should be within the replace menu and allow to select replacement with arrow keys and enter.

__Describe alternatives you've considered__

None.

__Additional context__

Tested via https://github.com/bpmn-io/bpmn-io-demo/pull/110

**Comment by nikku** [comment]
Good find. I think this is due to the way we bind global shortcuts in the demo. I cannot reproduce it in the sandbox: https://bpmn-io.github.io/bpmn-js-connectors-extension/?aa=1.
